.github/workflows/release.yaml

```diff
@@ -4,9 +4,13 @@ on:
   push:
     branches: [master]
 
+env:
+  FORCE_COLOR: 2
+  NODE: 14.x
+
 jobs:
   release:
-    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    if: "!contains(github.event.head_commit.message, 'skip ci') && github.repository == 'tancredi/fantasticon'"
     name: Release
 
     runs-on: ubuntu-18.04
```

Comment:
Forgot to ask @tancredi any specific reason you don't use latest here and in the other workflow?

Reply:
I prefer explicit versioning as it is more predictable - if any issues emerge in the future I'd rather be able to relate that to a change that was made, and this makes it easier to understand what environment has been used